looprpc/client.pb.gw.go

```diff
@@ -45,6 +48,23 @@ func request_SwapClient_LoopOut_0(ctx context.Context, marshaler runtime.Marshal
 
 }
 
+func local_request_SwapClient_LoopOut_0(ctx context.Context, marshaler runtime.Marshaler, server SwapClientServer, req *http.Request, pathParams map[string]string) (proto.Message, runtime.ServerMetadata, error) {
```

Comment:
We should also update the versions of the `grpc-gateway`, `genproto` and `grpc` libraries to reflect these changes. Otherwise we could have a divergence between the generated code and the runtime library. Optimally this would be caught by compiler errors but we never know if any silent behavior changes could break our code.

Reply:
Fixed. Only the `grpc-gateway` ref needed an update.